src/rpc/format/cbor/convert.rs

```diff
@@ -80,13 +132,38 @@ impl TryFrom<Cbor> for Value {
 						_ => Err("Expected a CBOR text data type"),
 					},
 					// A literal duration
-					TAG_DURATION => match *v {
+					TAG_STRING_DURATION => match *v {
 						Data::Text(v) => match Duration::try_from(v) {
 							Ok(v) => Ok(v.into()),
 							_ => Err("Expected a valid Duration value"),
 						},
 						_ => Err("Expected a CBOR text data type"),
 					},
+					// A custom [seconds: Option<u64>, nanos: Option<u32>] duration
+					TAG_DURATION => match *v {
```

Comment:
Can we call this `TAG_BINARY_DURATION`?

Reply:
Not really binary. But I do not have a better name.